spec/model/trial_user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :model do
+  it 'is valid with valid attributes' do
+    user1 = User.new(email: 'email@example.com', password: 'password')
+    expect(user1).not_to be_valid
```

Comment:
This is so confusing, you're test is `is valid with valid attributes` but you're expecting an invalid result

Reply:
Try to complete the attributes since this test is a happy path test